actors/evm/src/interpreter/stack.rs

```diff
@@ -147,6 +147,19 @@ fn test_stack_push_pop() {
     assert_eq!(stack.pop().unwrap(), 1);
 }
 
+#[test]
+fn test_stack_drop() {
+    let mut stack = Stack::new();
+    stack.push(1.into()).unwrap();
+    stack.push(2.into()).unwrap();
+    stack.drop().unwrap();
+    stack.drop().unwrap();
+    assert_eq!(
+        stack.drop().expect_err("expect underflow"),
+        ActorError::unchecked(EVM_CONTRACT_STACK_UNDERFLOW, "stack underflow".into())
```

Comment:
Let's not check specific error messages, that gets annoying pretty fast. Instead, just extract and test the exit code, that's the only thing observable by the outside.